packages/components/src/components/contract-card/contract-card-items/contract-card-body.jsx

```diff
@@ -80,7 +83,7 @@ const ContractCardBody = ({
                 toggleCancellationWarning={toggleCancellationWarning}
             />
         );
-    } else if (is_turbos) {
+    } else if (is_turbos && !is_open_positions) {
```

Comment:
@hamza-deriv let's have one else is_turbos, one component and jsut pass is_open_positions props inside TurbosOpenCardBody:
```
else if (is_turbos) {
    card_body = (
            <TurbosOpenCardBody
            is_open_positions={is_open_positions}
```

Reply:
We can use TurboCardBody and add classes based on the requirements and positions like `open-positions`, `recent-positions`